runtime/compiler/codegen/CodeGenGC.cpp

```diff
@@ -71,117 +68,125 @@ J9::CodeGenerator::createStackAtlas()
    //
    TR::Compilation *comp = self()->comp();
    TR::ResolvedMethodSymbol * methodSymbol = comp->getMethodSymbol();
-   int32_t slotIndex = 0;
 
-   intptrj_t stackSlotSize = TR::Compiler->om.sizeofReferenceAddress();
+   bool doLocalsCompaction = (self()->getLocalsIG() && self()->getSupportsCompactedLocals()) ? true : false;
```

Comment:
I'm sure you're doing a ton more work in this area, so perhaps we can fix this one on the next round. How about:

```
const bool doLocalsCompaction = self()->getLocalsIG() && self()->getSupportsCompactedLocals();
```